engine/server/engine/enclave_manager/enclave_manager.go

```diff
@@ -305,14 +313,19 @@ func (manager *EnclaveManager) Clean(ctx context.Context, shouldCleanAll bool) (
 	if len(successfullyRemovedArtifactIds) > 0 {
 		logrus.Infof("Successfully removed the enclaves")
 		sort.Strings(successfullyRemovedArtifactIds)
-		for _, successfullyRemovedEnclaveUuid := range successfullyRemovedArtifactIds {
-			enclaveName := manager.getEnclaveNameForEnclaveUuidUnlocked(successfullyRemovedEnclaveUuid)
+		for _, successfullyRemovedEnclaveUuidStr := range successfullyRemovedArtifactIds {
```

Comment:
Is `successfullyRemoved_Artifact_Ids ` expected here? isn't this about enclaves? 
Also, `successfullyRemovedEnclave_Uuid_Str` seem to be UUID, so maybe also rename this to xxxUUID instead